packages/react-native-aztec/android/build.gradle

```diff
@@ -7,9 +7,9 @@ buildscript {
         robolectricVersion = '3.5.1'
         jUnitVersion = '4.12'
         jSoupVersion = '1.10.3'
-        wordpressUtilsVersion = '2.3.0'
+        wordpressUtilsVersion = '110-13b4f06680ed8c145cbd09d63c4998dfe8b3b9de'
         espressoVersion = '3.0.1'
-        aztecVersion = 'v1.6.0'
+        aztecVersion = '1006-3fdbbc2086fb71d3750b01c013fa5cf6024c3142'
```

Comment:
The PRs that point these dependencies are the ones where Android API version is also being updated to 31:
- https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/110
- https://github.com/wordpress-mobile/AztecEditor-Android/pull/1006

Reply:
Solving this as Aztec now points to the latest version `v1.6.2`.